### DISCUSSION TITLE
Installing error!

### DISCUSSION THREAD

**Discussion by optimisticubermenzsch** [ohSystemmm/ohSystemmm-doties#1] [Q&A] [answered]
Hello! I have been trying to installing your dotfiles as in minimal and fresh archinstall. However, everytimes I try to install, it installed everything completely, I have tried essential packages with optional and with all the apps too. Choosing zsh, bash even foot. All possible outcomes. Everything installed perfectly, even grub theme also, however it failed to boot due to some error! I am unsure whether it's a dotfiles error or I have done my base config of archinstall wrong. I have installed arch with btrfs and with compression, no disk encryption, pipewire, tried both linux main and zen kernel, however the same output. 

For context, I will upload the error I am getting. I hope, you fix the issue if it is. And if it is my mistake of installation, please correct me and provide me a proper way to install it. 

For additional context, I have intel cpu with integrated graphics drives, none dedicated one. 

This is the continuous error I am getting after every successful install. 

- [:: running early hook [udeu]

Starting systemd-udeud version 256.6-1-arch

:: running hook [udev] :: Triggering uevents...

:: running hook [keymap]

:: Loading keymap...done.

:: performing fsck on 'UUID=c09cd0ef-aae3-4a1e-a5c7-c52bc67c2ecc' :: mounting 'UUID=c09cd0ef-aae3-4a1e-a5c7-c52bc67c2ecc' on real root

mount: /new_root: unknown filesystem type 'ext4'

dmesg (1) may have more information after failed mount system call. ERROR: Failed to mount 'UUID=c09cd0ef-aae3-4a1e-a5c7-c52bc67c2ecc' on real root

You

are now being dropped into an emergency shell. sh: can't access tty: job control turned off

[rootfs ~]# [ 165.310675] usb 1-1: device descriptor read/64, error -71

[175.357365] usb usb1-port1: Cannot enable. Maybe the USB cable is bad? ] 

I am expecting you will guide me to install it perfectly. Thankyou in advance!

**Comment by ohSystemmm** [comment, marked as answer]
Hello, I am glad that you're interested in using them. 
The problem is probably in /etc/default/grub. Edit the rootfstype in line 15 - try replacing it with btrfs. Save + update grub and run mkinitcpio -P. I think it should fix it. Let me know it it worked! 

Thanks for choosing my dotfiles!

Greetings
ohSystemmm

![image](https://github.com/user-attachments/assets/814eab30-e036-498c-8c62-ce0366e6f3ee)
